frontend/src/index.jade

```diff
@@ -148,10 +158,10 @@ html
                 span.select(v-bind:title="getTotalFileBlankLineInfo()") All:&nbsp;
                 span.loc(v-bind:title="getTotalFileBlankLineInfo()") {{ totalLineCount }}&nbsp;
                 span.bloc(v-bind:title="getTotalFileBlankLineInfo()") ({{ totalBlankLineCount }})
-              template(v-for="type in Object.keys(filesLinesObj)")
+              template(v-for="type in Object.keys(info.filesLinesObj)", v-if="info.filesLinesObj[type] > 0")
```

Comment:
dun use `v-if` and `v-for` in the same object, and do `v-bind:key` for each `v-for` obj.

do `template(v-if='...')`

Reply:
I've filtered ```filesLinesObj``` by iterating over a computed property ```filesExistingLinesObj``` instead as per this: https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential